pkg/registry/core/serviceaccount/strategy_test.go

```diff
@@ -0,0 +1,123 @@
+package serviceaccount
+
+import (
+	"context"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	api "k8s.io/kubernetes/pkg/apis/core"
+)
+
+func TestNamespaceScoped(t *testing.T) {
```

Comment:
This looks like AI generated unit tests for things unrelated to this PR.  Lets focus only on adding tests for things you changed (applies to other tests as well).